centreon/src/Core/Dashboard/Application/UseCase/PartialUpdateDashboard/PartialUpdateDashboardRequest.php

```diff
@@ -24,22 +24,222 @@
 namespace Core\Dashboard\Application\UseCase\PartialUpdateDashboard;
 
 use Core\Common\Application\Type\NoValue;
+use Core\Dashboard\Application\UseCase\PartialUpdateDashboard\Request\PanelLayoutRequestDto;
 use Core\Dashboard\Application\UseCase\PartialUpdateDashboard\Request\PanelRequestDto;
 use Core\Dashboard\Application\UseCase\PartialUpdateDashboard\Request\RefreshRequestDto;
+use Core\Dashboard\Application\UseCase\PartialUpdateDashboard\Request\ThumbnailRequestDto;
+use Core\Dashboard\Infrastructure\Model\RefreshTypeConverter;
+use Symfony\Component\Validator\Constraints as Assert;
 
 final class PartialUpdateDashboardRequest
```

Comment:
I don't understand this class, normally it's a request class so a dto with only scalar values. I see several methods in this class. I don't understand the implmentation. Can you explain me please ? Thanks.

Reply:
See previous explanation

Reply:
see https://github.com/centreon/centreon/pull/5150#discussion_r1824085249

Reply:
We have found a solution, thanks for your change ;)